package.json

```diff
@@ -35,8 +35,8 @@
     "doctest": "0.17.x",
     "eslint": "6.8.x",
     "eslint-plugin-markdown": "1.0.x",
-    "mocha": "5.x.x",
```

Comment:
I recall something preventing us from upgrading to `mocha@6.x.x`. Do you remember what it was, @Avaq? My guess is that it was Istanbul. If this is true, switching from Istanbul to nyc in #18 enabled this upgrade.

Reply:
I don't remember what the reason was, David. But that sounds like it might have been it.